example/src/main/java/com/stripe/example/activity/PaymentActivity.java

```diff
@@ -72,44 +92,36 @@ public void onClick(View view) {
         initListView();
     }
 
-    public void saveCreditCard() {
-        String cardNumber = cardNumberEditText.getText().toString();
-        String cvc = cvcEditText.getText().toString();
-
-        int expMonth = getIntegerFromSpinner(monthSpinner);
-        int expYear = getIntegerFromSpinner(yearSpinner);
-
-        String currency = getCurrency();
-        Card cardToSave = new Card(cardNumber, expMonth, expYear, cvc);
-        cardToSave.setCurrency(currency);
-
-        if (validateCard(cardToSave)) {
-            startProgress();
-            new Stripe().createToken(
-                    cardToSave,
-                    PUBLISHABLE_KEY,
-                    new TokenCallback() {
-                        public void onSuccess(Token token) {
-                            addToList(token);
-                            finishProgress();
-                        }
-                        public void onError(Exception error) {
-                            handleError(error.getLocalizedMessage());
-                            finishProgress();
-                        }
-                    });
+    @Override
+    protected void onDestroy() {
+        super.onDestroy();
+        if (compositeSubscription != null) {
```

Comment:
This is the magic of composite subscriptions -- the one knock against rx java is that it makes it very easy to leak `Context` if you're not careful about unsubscribing and nulling things out.